packages/@aws-cdk/aws-eks/test/bucket-pinger/bucket-pinger.ts

```diff
@@ -1,30 +1,27 @@
-import * as ec2 from '@aws-cdk/aws-ec2';
 import * as iam from '@aws-cdk/aws-iam';
 import * as lambda from '@aws-cdk/aws-lambda';
 import { CustomResource, Token, Duration } from '@aws-cdk/core';
 import * as cr from '@aws-cdk/custom-resources';
 import { Construct } from 'constructs';
 
-export interface PingerProps {
-  readonly securityGroup?: ec2.SecurityGroup;
-  readonly vpc?: ec2.IVpc;
-  readonly subnets?: ec2.ISubnet[];
+export interface BucketPingerProps {
+  readonly bucketName: string;
 }
 export class BucketPinger extends Construct {
 
   private _resource: CustomResource;
 
-  constructor(scope: Construct, id: string, props: PingerProps) {
+  constructor(scope: Construct, id: string, props: BucketPingerProps) {
     super(scope, id);
 
     const func = new lambda.Function(this, 'Function', {
       code: lambda.Code.fromAsset(`${__dirname}/function`),
       handler: 'index.handler',
       runtime: lambda.Runtime.PYTHON_3_9,
-      vpc: props.vpc,
-      vpcSubnets: props.subnets ? { subnets: props.subnets } : undefined,
-      securityGroups: props.securityGroup ? [props.securityGroup] : undefined,
```

Comment:
Removed these because they are not required. The pinger simply makes a HEAD request to a bucket, it doesn't need to be in the VPC of the cluster. Also, [Lambda's attached to a VPC take a much longer time to delete](https://stackoverflow.com/questions/47957820/lambda-in-vpc-deletion-takes-more-time).